src/morphs.jl

```diff
@@ -135,10 +135,10 @@ function save_morph_polygons!(action::Action, from_func::Function, to_func::Func
     smallest_i = 1
     smallest_distance = typemax(Float64)
 
-    for i in 1:length(from_poly)
+    for i = 1:length(from_poly)
         overall_distance = 0.0
         for j = 1:length(from_poly)
-            p1 = from_poly[(j+i-1) % length(from_poly) + 1]
+            p1 = from_poly[(j+i-1)%length(from_poly)+1]
```

Comment:
I actually hate that there is no space around the %. 😄  Any idea why? Are there options to change that behavior?

Reply:
I don't know why, but there is a way to disable this using the kwarg, `whitespace_ops_in_indices` set to true.